kinder/pkg/cluster/manager/create.go

```diff
@@ -113,11 +113,23 @@ func CreateCluster(clusterName string, options ...CreateOption) error {
 	handleErr := func(err error) error {
 		// In case of errors nodes are deleted (except if retain is explicitly set)
 		if !flags.retain {
-			ctx := kindcluster.NewContext(clusterName)
-			ctx.Delete()
+			if c, err := status.FromDocker(clusterName); err != nil {
+				log.Error(err)
+			} else {
```

Comment:
i don't think this else branch is needed here.

Reply:
Uhmmm, the else branch is the one doing the actual deletion in case FromDocker does not return errors